### PR TITLE
feat: HCaptcha Middleware

### PR DESCRIPTION
**Pull request by bndrmrtn**
## Summary by CodeRabbit

- **New Features**
	- Enhanced security by introducing HCaptcha middleware to verify tokens and prevent robot submissions in Go applications.
	- Added a GitHub Actions workflow to automate testing of the HCaptcha functionality.

- **Documentation**
	- Updated the main README and provided detailed documentation for the HCaptcha middleware.

- **Tests**
	- Implemented tests to validate the functionality of the HCaptcha middleware integration.

- **Chores**
	- Configured release drafter settings to manage HCaptcha releases effectively.

### CODE REVIEW

**Comment by ReneWerner87** [comment]
- [ ] add this new middleware in the root readme
- [ ] add workflows for this new middleware(dependabot, test, release-drafter ... )
- [ ] add readme for the middleware itself

**Comment by bndrmrtn** [comment]
> * [ ]  add this new middleware in the root readme
> 
>     * [ ]  add workflows for this new middleware(dependabot, test, release-drafter ... )
> 
>     * [ ]  add readme for the middleware itself

I added it to the root README and I created a README file for it, I tried to enable dependabot, but I don't know how to excatly enable workflows for this pull request. I never did it before, that's new for me ://

**Comment by gaby** [comment]
@bndrmrtn Copy one of the test workflows for the other contrib middlewares, and adapt it to this one.

**Comment by bndrmrtn** [comment]
I think I did it, right? 🤔

**Comment by ReneWerner87** [comment]
will check it tomorrow

File: README.md [hunk 1]
```diff
@@ -34,3 +34,4 @@ Repository for third party middlewares with dependencies.
 * [Socket.io](./socketio/README.md) <a href="https://github.com/gofiber/contrib/actions?query=workflow%3A%22Test+socketio%22"> <img src="https://img.shields.io/github/actions/workflow/status/gofiber/contrib/test-socketio.yml?branch=main&label=%F0%9F%A7%AA%20&style=flat&color=75C46B" /> </a>
 * [Swagger](./swagger/README.md) <a href="https://github.com/gofiber/contrib/actions?query=workflow%3A%22Test+swagger%22"> <img src="https://img.shields.io/github/actions/workflow/status/gofiber/contrib/test-swagger.yml?branch=main&label=%F0%9F%A7%AA%20&style=flat&color=75C46B" /> </a>
 * [Websocket](./websocket/README.md) <a href="https://github.com/gofiber/contrib/actions?query=workflow%3A%22Test+websocket%22"> <img src="https://img.shields.io/github/actions/workflow/status/gofiber/contrib/test-websocket.yml?branch=main&label=%F0%9F%A7%AA%20&style=flat&color=75C46B" /> </a>
+* [HCaptcha](./hcaptcha/README.md)
```
Comment (ReneWerner87) [inline README.md:37]: pls consider the order and also add the badges like the others

**Review by ReneWerner87 — CHANGES_REQUESTED** [review]
- [x] add release drafter workflow and config
- [x] add section to the dependabot.yml

**Comment by ReneWerner87** [comment]
pls add a section like this
https://github.com/gofiber/contrib/blob/f6a1c31d58164c7f7339a9b96222a73a66f4c01e/.github/dependabot.yml#L101-L106

and copy the release drafter yaml´s

**Comment by ReneWerner87** [comment]
@bndrmrtn pls don't create useless pull requests
Screenshot_2024-04-18-19-29-02-30_40deb401b9ffe8e1df2f1cc5ba480b12

**Comment by ReneWerner87** [comment]
Here some article for new contributers

https://github.com/firstcontributions/first-contributions

**Comment by bndrmrtn** [comment]
> @bndrmrtn pls don't create useless pull requests
> Screenshot_2024-04-18-19-29-02-30_40deb401b9ffe8e1df2f1cc5ba480b12
> 

yeah, really sorry 😅

File: README.md (hunk 1 above)
Comment (ReneWerner87) [reply README.md]: fixed with https://github.com/gofiber/contrib/pull/1071/commits/f5b94a8919743e39ccf0c7e26b4e6cab6fe7e625

**Comment by ReneWerner87** [comment]
> * [x]  add release drafter workflow and config
> * [x]  add section to the dependabot.yml

fixed with https://github.com/gofiber/contrib/pull/1071/commits/f5b94a8919743e39ccf0c7e26b4e6cab6fe7e625

**Comment by ReneWerner87** [comment]
@gaby @sixcolors @efectn can you check

**Comment by ReneWerner87** [comment]
@bndrmrtn https://github.com/gofiber/contrib/releases/tag/hcaptcha%2Fv0.1.0

**Comment by bndrmrtn** [comment]
Thanks! 🤩
Also I really appriciate your help, cuz I'm new to contributing code, that was my first. It feels great that my code is merged into my favourite framework. 😁
I hope it will help some devs integrating with this captcha.

**Pull request merged**